spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -240,17 +215,16 @@
 """
 
 
-def getHeader(productName, options, orgCACert, isRpmYN, pubname, apachePubDirectory):
+def getHeader(productName, options, orgCACert, pubname, apachePubDirectory):
```

Comment:
This function is only expected to be called from `spacewalk/certs-tools/rhn_bootstrap.py`, right? Just double checking that we don't expect anyone to call this function and pass too many args after this change.